src/libs/ReportUtils.js

```diff
@@ -2011,6 +2011,25 @@ function getChatByParticipants(newParticipantList) {
     });
 }
 
+/**
+ * Attempts to find a report in onyx with the provided list of participants. Does not include threads
+ * @param {String} accountID
+ * @returns {boolean|undefined}
```

Comment:
```suggestion
 * @returns {boolean}
```

I guess this only return a boolean, right?